storage/time-series/influxdb/retriever/README.md

```diff
@@ -0,0 +1,159 @@
+
+## Running the service
+
+### Configurations
+
+Before running the **InfluxDB Storer** service within your environment, make sure you configure the
+environment variables to match your needs.
+
+You can select the configuration file via the `RETRIEVER_APP_USER_CONFIG_FILE` variable. Its default value
+is `production.conf`. Check the [config directory](./config) for the user configurations that are
+available by default.
+
+For more information about the usage of the configuration files and environment variables, check the
+__ConfigManager__ module in our [Microservice SDK](https://github.com/dojot/dojot-microservice-sdk-js).
+You can also check the [ConfigManager environment variables documentation](https://github.com/dojot/dojot-microservice-sdk-js/blob/master/lib/configManager/README.md#environment-variables) for more details.
+
+In short, all the parameters in the next sections are mapped to environment variables that begin
+with `RETRIEVER_`. You can either use environment variables or configuration files to change their values.
+You can also create new parameters via environment variables by following the fore mentioned
+convention.
+
+#### General Configurations
+
+| Key | Purpose | Default Value | Valid Values | Environment variable
+| --- | ------- | ------------- | ------------ | --------------------
+| log.console.level | Console logger level | info | info, debug, error, warn | RETRIEVER_LOG_CONSOLE_LEVEL
+| log.file | Enables logging on file (location: /var/log/influxdb-retriever-logs-%DATE%.log) | false | boolean  | RETRIEVER_LOG_FILE
+| log.file.level  | Log level to log on files | debug | string  | RETRIEVER_LOG_FILE_LEVEL
+| log.verbose | Whether to enable logger verbosity or not | false | boolean | RETRIEVER_LOG_VERBOSE
+| express.trustproxy | Enables reverse proxy support  | true | boolean | RETRIEVER_EXPRESS_TRUSTPROXY
+| paginate.default.max.limit |  Sets maximum and default the item numbers on each page  | 256 | integer | RETRIEVER_PAGINATE_DEFAULT_MAX_LIMIT
+
+#### Server Configurations
+
+| Key | Purpose | Default Value | Valid Values | Environment variable
+| --- | ------- | ------------- | ------------ | --------------------
+| server.host | Server address | 0.0.0.0 | string  | RETRIEVER_SERVER_HOST
+| server.port | Sever Port  | 3000 | integer  | RETRIEVER_SERVER_PORT
+| server.ca | File path to list of supplied CAs. If passed enable TLS  | none | path  | RETRIEVER_SERVER_CA
+| server.cert | File path to  certificate.  | none | path| RETRIEVER_SERVER_CERT
+| server.key | File path to key certificate. | none | path |  RETRIEVER_SERVER_KEY
+| server.reject.unauthorized | If true, the server certificate is verified against the list of supplied CAs. | none | boolean | RETRIEVER_SERVER_REJECT_UNAUTHORIZED
+| server.request.cert | Whether to authenticate the remote peer by requesting a certificate. Clients always request a server certificate. | none | boolean | RETRIEVER_SERVER_REQUEST_CERT
```

Comment:
No, but it can be configured to use https or https mutual, the sdk offers this possibility ... It would only be necessary to make the settings to use it.

Reply:
OK